pyproject.toml

```diff
@@ -1,3 +1,21 @@
 [tool.ruff]
-line-length = 88
 target-version = "py312"
+line-length = 88
+
+[tool.ruff.format]
+docstring-code-line-length = 100
```

Comment:
Would `dynamic` be a better setting for the code-line-length? The documentation is not super-clear to me, but it includes:
>The dynamic setting ensures that even when code examples are found inside indented docstrings, the line length limit configured for the surrounding Python code will not be exceeded.

https://docs.astral.sh/ruff/formatter/#docstring-formatting